### PR TITLE
Life, death and repeat

### PR DESCRIPTION
**Pull request by LidiaYoshino**
## This PR adds:
- life (health)
- death
- restart button
- state/info structure

The screen doesn't look good, but it works hahaha
Will revamp graphics when I have more time to work on it. Focusing on making it work first and then I'll come back and improve it.

I know this is not the best way to manage state, but I wanted to understand better how info gets passed from one component to their children ([Lifted up state](https://reactjs.org/docs/lifting-state-up.html)). I intend on maintaining the components nesting simple, with maybe only one more layer, we'll see if it is doable or if it will become a state caos...

## Testing:

- Follow read me to get repo running locally
- Access http://localhost:3000/
- Click on `Start!` link
- Wait till Tamago dies T^T
- Click on `Restart` button

### CODE REVIEW

File: src/Game.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        hunger: 100,
+        fun: 100,
+        hygiene: 100,
+        sleep: 100,
+      }
+    };
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.started === false && this.state.started === true) {
```
Comment (willsgimenes) [inline src/Game.jsx]: Hey @LidiaYoshino Do you intend to use hooks? If you, maybe useEffect here can be useful :-)

File: src/Game.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  tick() {
+    if (this.state.health > 0) {
+      this.setState({
+        health: this.state.health - 1,
+      });
+    } else {
+      clearInterval(this.timerID);
+    }
+  }
```
Comment (willsgimenes) [inline src/Game.jsx]: @LidiaYoshino you can simplify this function and extract this to another place where you can unit test it.

File: src/containers/PlayScreen/index.js [hunk 3]
```diff
@@ -3,11 +3,24 @@ import React from 'react';
...
+}
+
+const PlayScreen = ({ health }) => (
   <Header>
     <Title>
-      Play!
+      Tamago
+    </Title>
+    <Title>
+      { health <= 0 ? '( x _ x )' : '( ^ _ ^ )' }
```
Comment (willsgimenes) [inline src/containers/PlayScreen/index.js]: @LidiaYoshino you can extract this statement to a constant and then use it here :-)

File: src/Game.jsx (hunk 1 above)
Comment (LidiaYoshino) [reply src/Game.jsx]: I want to work without hooks first, to understand the basics. I'll refactor to add them later or use them in a different project

**Review by rmsphd — APPROVED** [review]
You need to add unit tests but I believe that you can add more tests in the future version

**Review by willsgimenes — APPROVED** [review]
Keep in mind some of the comments and maybe you can open some issues inside the project to address them :-)

**Pull request merged**